.vscode/tasks.json

```diff
@@ -54,6 +54,46 @@
         }
       ]
     },
+    {
+      "label": "Run Pester (WSL)",
+      "dependsOn": "publish",
+      "type": "shell",
+      "command": "pwsh",
+      "args": ["-Command", "Invoke-Pester"],
+      "options": {
+        "shell": {
+          "executable": "pwsh",
+          "args": ["-c"]
+        }
+      },
```

Comment:
I don't think you need to define this, on mac/Linux it doesn't make sense to run this task anyway

Reply:
It makes my life easier on Windows ;)

Reply:
I meant the `options` above this line - on Windows they will be overridden by the `windows` section below, and on mac/Linux you can just use `Run Pester`

Reply:
Yeah but then I can't run it on "normal" PowerShell

Reply:
Sorry, I'm not following. This task is called "Run Pester (WSL)" so I think it is for explicitely launching Pester in WSL on Windows. For running Pester in "normal" there's the normal "Run Pester" task?

Reply:
Yeah, there's 2 tasks (well, 4 actually).

I'm after an easy way to:

- Run in PowerShell on Windows
- Run in PowerShell on Linux

So that's what those two tasks are. Now the WSL one won't work if you're not on Windows but I can't think of any other way to do it

Reply:
You can remove the `windows` field and directly only define `options` with the Windows-specific options

Reply:
It's not possible to launch two different kinds of shells under Windows. `Run Pester` needs to launch `powershell.exe` on Windows, but there's no way to go from `powershell.exe` to `pwsh` under WSL without launching `bash.exe` first, which is what the 2nd one does.

Reply:
I understand that, but you shouldn't need L65 for that, because you override it in L72 anyway. You should be able to just directly use `bash.exe` in L65 and remove the whole `windows` bag.

Reply:
Not in my testing so far. First you need to launch `bash.exe` and then you need to run `pwsh` with specific arguments to _then_ run `Invoke-Pester`. It's a bit convoluted since it's shells in shells.